hummingbird/ml/_parse.py

```diff
@@ -19,6 +21,13 @@
 from ._utils import sklearn_installed
 from .supported import get_sklearn_api_operator_name, get_onnxml_api_operator_name
 
+if sklearn_installed():
```

Comment:
Is this ever false since it's a requirement of the install for HB?

Reply:
yep, let me remove it. (BTW do you remember why sklearn has always to be installed?)

Reply:
we tried to remove it, but then we realized that xgb,lgbm,etc all take dependencies on it.
Without at least [skl | onnx] installed, HB is unusable, so we decided to leave it